crates/build/re_types_builder/src/docs.rs

```diff
@@ -417,6 +417,13 @@ mod doclink_translation {
                 }
             }
             Target::WebDocsMarkdown => {
+                let kind_and_type = format!("{kind}.{type_name}");
+
+                // TODO(andreas): We don't show blueprint components & archetypes in the web docs yet.
+                if scope == "blueprint" && (kind == "components" || kind == "archetypes") {
+                    return Ok(kind_and_type);
+                }
```

Comment:
I've been meaning to bring that up a bunch of times, and always forget... Why are we still not showing those? :thinking:

Reply:
as per short conversation on zoom: those types don't really behave like the others as they don't have any effect when logging them to the store. We'd need to handle them very separately